agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/AgentSpanExporter.java

```diff
@@ -58,6 +58,9 @@ public CompletableResultCode export(Collection<SpanData> spans) {
     }
     for (SpanData span : spans) {
       logger.debug("exporting span: {}", span);
+
+      // TODO apply exception sampling overrides to span events!
```

Comment:
not ready for review yet, need to do this still

Reply:
nevermind, sampling overrides are already applied to span events, there was just an existing bug in it